### PR TITLE
[niina] Make FieldTest for saiki()

### PR DESCRIPTION
**Pull request by peria**
Many functions for field modification are modularized, we can and we must test them for future refactroing.

### CODE REVIEW

**Comment by peria** [comment]
PTL.
Or, should I add more tests for other public methods in this CL?

**Comment by mayah** [comment]
I prefer a large number of small patches to a small number of large patches.

File: src/cpu/test_lockit/field_test.cc [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        {1, 11, 2}, {2, 11, 2}
+    };
+
+    // We can re-use |point| array.
+    int point[6][12] {};
+    for (const TestData& data : data_set) {
+        int field[6][kHeight];
+
+        int num = 0;
+        toTLField(cf, field);
```
Comment (mayah) [inline src/cpu/test_lockit/field_test.cc]: Move this out of the loop?

File: src/cpu/test_lockit/field_test.cc [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    }
+
+    int count_point = 0;
+    for (int x = 0; x < 6; ++x) {
+        for (int y = 0; y < 12; ++y) {
+            if (point[x][y])
+                ++count_point;
+        }
+    }
+    EXPECT_EQ(26, count_point);
```
Comment (mayah) [inline src/cpu/test_lockit/field_test.cc]: could you accumulate expect_num in the loop, and use it here instead of 26?

Comment (peria) [inline src/cpu/test_lockit/field_test.cc]: done

File: src/cpu/test_lockit/field_test.cc (hunk 1 above)
Comment (peria) [inline src/cpu/test_lockit/field_test.cc]: done

**Comment by mayah** [comment]
LGTM

**Pull request merged**